dev/components/recordPopup/component-mobile.js

```diff
@@ -44,5 +46,17 @@
         vm.videoName = null;
       });
     };
+
+
+    vm.sendRecordClick = function (name) {
+      name = name || 'no name';
+      uploadService.sendFile(
+        '/api/video',
+        vm.recordedData.video,
+        name + '.wmv'
```

Comment:
This extension is wrong. Please verify the information https://github.com/collab-project/videojs-record